src/api/logs.coffee

```diff
@@ -0,0 +1,26 @@
+logger = require 'winston'
+moment = require 'moment'
+Q = require 'q'
+authorisation = require './authorisation'
+utils = require "../utils"
+
+exports.getLogs = ->
```

Comment:
It might be good to think a bit about how this endpoint will be used from the console with regard to filters. Two uses I can think of would be to filter by date range as well as filter by log level.

In addition it would be cool if the console page continually updates with the latest log entries - I think for all we would need is to supply the from date, leaving the until as the latest time. (the from would be the console's last sync date - this is how the transaction log updates work)